redoctober.go

```diff
@@ -805,6 +900,99 @@ var indexHtml = []byte(`<!DOCTYPE html>
 					}
 				});
 			});
+			// Create an order
+			$('body').on('submit', 'form#order', function(evt){
+				evt.preventDefault();
+				var $form = $(evt.currentTarget),
+					data = serialize($form);
+
+				submit( $form, {
+					data : data,
+					success : function(d){
+					d = JSON.parse(window.atob(d.Response));
+					$form.find('.feedback').empty().append(
```

Comment:
I understand keeping `.empty().append()` for consistency sake, especially because then it have the same structure as forms like `create-user` where it just does a `.append()`.

As for the security, `.html(value)` actually just ends up calling `.empty().append(value)` (see [source code](https://github.com/jquery/jquery/blob/22449eb/src/manipulation.js#L425)) so I believe they're equally susceptible to XSS